s3/pkg/db/drivers/mongo/listobjects.go

```diff
@@ -112,26 +119,32 @@ func (ad *adapter) CountObjects(in *pb.ListObjectsRequest, out *utils.ObjsCountI
 		filt = in.Filter[common.KObjKey]
 	}
 
+	m := bson.M{
+		utils.DBKEY_OBJECTKEY:    bson.M{"$regex": filt},
+		utils.DBKEY_INITFLAG:     bson.M{"$ne": "0"},
+		utils.DBKEY_DELETEMARKER: bson.M{"$ne": "1"},
+	}
```

Comment:
it's better to use constants here

Reply:
Those keys defined by us used constants, like DBKEY_OBJECTKEY, DBKEY_INITFLAG, etc. But those regular expression keys, like '$regex', '$ne' does not use constants, because they are specific and more intuitive.